src/couch/src/couch_sup.erl

```diff
@@ -168,3 +169,8 @@ write_file(FileName, Contents) ->
             couch_log:error("Failed ot write ~s :: ~s", Args),
             throw({error, Reason})
     end.
+
+start_password_server() ->
+    couch_log:info("Password Server Process is starting.~n", []),
+    couch_password_server:start_link(),
+    ok.
```

Comment:
If we intend this to be properly supervised we should add it to the supervisor as a child. `couch_primary_services` might be a better place for it, right before the couch_server instances are added.

Reply:
So, put it into Children of `couch_primary_services`, or where to add?